spec.md

```diff
@@ -3489,17 +3496,18 @@ type_hardware =
 type_ground =  type_ground_nowidth | type_ground_width ;
 
 type_ground_nowidth =
-    "Clock"
-  | "Reset"
-  | "AsyncReset" ;
+    "Clock" ;
 
 type_ground_width =
     "UInt" , [ width ]
   | "SInt" , [ width ]
-  | "Analog" , [ width ] ;
+  | "Analog" , [ width ] 
+  | "Reset" , [ resetspec ] ;
 
 width = "<" , int , ">" ;
 
+resetspec = "<" , ( "sync" | "async" | "syncasync" ) ,  "," , ( "high | "low" ) , ">" ;
```

Comment:
Could we name the categories for these parameters and introduce two simple nonterminals?

For instance:

```
resetspec = "<" , reset_synchronicity ,  "," , reset_activation , ">" ;

reset_synchronicity = "sync" | "async" | "syncasync" ;

reset_activation = "high | "low" ;
```